pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -264,6 +266,7 @@ func (m *manager) Allocate(p *v1.Pod, c *v1.Container) error {
 		return err
 	}
 
+	klog.V(4).InfoS("CPUs allocated by cpumanager successfully", "pod", klog.KObj(p), "podUID", p.UID, "containerName", c.Name)
```

Comment:
same reasoning to drop "succesfully"
I'd rephrase as `Allocated exclusive CPUs`.
Even better: move the log line inside the `policy` so we can also log what was allocated